examples/annotations.openapi3.json

```diff
@@ -3061,6 +3061,9 @@
                 ],
                 "description": "The number of entities in the collection. Available when using the [$count](http://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part1-protocol.html#sec_SystemQueryOptioncount) query option."
             },
+            "type": {
+                "type": "string"
+            },
```

Comment:
Probably better to not have this „reuse schema“, or only have it when it isn’t needed.

Reply:
I found this function `deleteUnreferencedSchemas` which has the desired effect. But it also removes a lot of other unused schemas causing a bunch of tests to fail. Fixable though.

But I assume `deleteUnreferencedSchemas` wasn't used for a reason?

Reply:
Reason is that there can be CSDL files with schemas for reusable types and no entity container. The types are only referenced from other files, not from within the same file.

Consequently "delete unreferenced schemas" is not on by default.

Easiest fix here is not to define a reuse schema and generate `@type` directly with `type: "string"`.